Libraries/Hotcakes.Commerce/BusinessRules/OrderTasks/StartPaypalExpressCheckout.cs

```diff
@@ -101,12 +106,14 @@ public override bool ProcessCheckout(OrderTaskContext context)
 
                         if (address.CountryData != null)
                         {
-							var itemsTotalWithoutTax = context.Order.TotalOrderAfterDiscounts;
+                            var ISOCode = address.CountryData.IsoCode;
+
+                            var itemsTotalWithoutTax = context.Order.TotalOrderAfterDiscounts;
 							if (context.HccApp.CurrentStore.Settings.ApplyVATRules)
 							{
 								itemsTotalWithoutTax -= context.Order.ItemsTax;
 							}
-							string itemsTotal = itemsTotalWithoutTax.ToString("N", CultureInfo.InvariantCulture);
+                            string itemsTotal = itemsTotalWithoutTax.ToString("N", CultureInfo.InvariantCulture);
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.